### ISSUE TITLE
Exception when moving a node to the root of the classification scheme or to another node (sometimes)

### ISSUE THREAD

**Issue by jmaferreira** [keeps/roda-in#326]
```
Cannot get the type of a change that neither inserts nor deletes anything.
	org.fxmisc.richtext.model.TextChange.<init>(TextChange.java:31)
	org.fxmisc.richtext.model.RichTextChange.<init>(RichTextChange.java:6)
	org.fxmisc.richtext.model.ReadOnlyStyledDocument.lambda$null$7(ReadOnlyStyledDocument.java:274)
	org.reactfx.util.Tuple2.map(Tuple2.java:31)
	org.fxmisc.richtext.model.ReadOnlyStyledDocument.lambda$replace$8(ReadOnlyStyledDocument.java:271)
	org.reactfx.util.Tuple2.map(Tuple2.java:31)
	org.fxmisc.richtext.model.ReadOnlyStyledDocument.replace(ReadOnlyStyledDocument.java:270)
	org.fxmisc.richtext.model.ReadOnlyStyledDocument.replace(ReadOnlyStyledDocument.java:260)
	org.fxmisc.richtext.model.GenericEditableStyledDocumentBase.setStyleSpans(GenericEditableStyledDocumentBase.java:154)
	org.fxmisc.richtext.model.SimpleEditableStyledDocument.setStyleSpans(SimpleEditableStyledDocument.java:7)
	org.fxmisc.richtext.GenericStyledArea.setStyleSpans(GenericStyledArea.java:1281)
	org.roda.rodain.ui.inspection.InspectionPane.lambda$createMetadataTextBox$1(InspectionPane.java:291)
	org.reactfx.value.ChangeListenerWrapper.accept(Val.java:786)
	org.reactfx.util.AbstractReducingStreamNotifications.lambda$head$0(NotificationAccumulator.java:248)
	org.reactfx.ObservableBase.notifyObservers(ObservableBase.java:68)
	org.reactfx.SuspendableBase.resume(SuspendableBase.java:64)
	org.reactfx.CloseableOnceGuard.close(Guard.java:49)
	org.reactfx.MultiGuard.close(Guard.java:83)
	org.reactfx.Suspendable$1.resumeSource(Suspendable.java:118)
	org.reactfx.Suspendable$1.suspendSource(Suspendable.java:104)
	org.reactfx.util.NonAccumulativeStreamNotifications.lambda$head$0(NotificationAccumulator.java:134)
	org.reactfx.ObservableBase.notifyObservers(ObservableBase.java:68)
	org.reactfx.ObservableBase.notifyObservers(ObservableBase.java:57)
	org.reactfx.ProperEventStream.emit(ProperEventStream.java:18)
	org.reactfx.EventStreams$3.lambda$observeInputs$0(EventStreams.java:105)
	org.reactfx.value.ChangeListenerWrapper.accept(Val.java:786)
	org.reactfx.util.AbstractReducingStreamNotifications.lambda$head$0(NotificationAccumulator.java:248)
	org.reactfx.ObservableBase.notifyObservers(ObservableBase.java:68)
	org.reactfx.ObservableBase.notifyObservers(ObservableBase.java:57)
	org.reactfx.value.ValBase.invalidate(ValBase.java:32)
	org.reactfx.SuspendableBoolean.release(SuspendableBoolean.java:24)
	org.reactfx.CloseableOnceGuard.close(Guard.java:49)
	org.reactfx.Suspendable.suspendWhile(Suspendable.java:49)
	org.fxmisc.richtext.model.GenericEditableStyledDocumentBase.update(GenericEditableStyledDocumentBase.java:218)
	org.reactfx.util.Tuple3.exec(Tuple3.java:43)
	org.fxmisc.richtext.model.GenericEditableStyledDocumentBase.replace(GenericEditableStyledDocumentBase.java:125)
	org.fxmisc.richtext.model.SimpleEditableStyledDocument.replace(SimpleEditableStyledDocument.java:7)
	org.fxmisc.richtext.GenericStyledArea.replace(GenericStyledArea.java:1302)
	org.fxmisc.richtext.GenericStyledArea.replaceText(GenericStyledArea.java:1297)
	org.fxmisc.richtext.model.EditActions.replaceText(EditActions.java:147)
	org.roda.rodain.ui.inspection.InspectionPane.lambda$updateTextArea$54(InspectionPane.java:2072)
	com.sun.javafx.application.PlatformImpl.lambda$null$172(PlatformImpl.java:295)
	java.security.AccessController.doPrivileged(Native Method)
	com.sun.javafx.application.PlatformImpl.lambda$runLater$173(PlatformImpl.java:294)
	com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
```
See example:

<img width="1672" alt="screen shot 2018-09-25 at 13 57 05" src="https://user-images.githubusercontent.com/3181302/46015778-0899ee80-c0cb-11e8-8c43-7c87d6f5a191.png">

**Comment by luis100** [comment]
That's a known bug in the richtextfx 0.7-M4 and 0.7-M5 releases. Currently using 0.7-M4.
Current latest version in Maven is [0.9.1](https://search.maven.org/artifact/org.fxmisc.richtext/richtextfx/0.9.1/jar) But this may break more things...

**Comment by luis100** [comment]
Blindly updated version of dependency in pom and compiled the attached binary.
Please test it, if it works we might consider pushing this into master. (renamed attachment with .zip due to GitHub restrictions).


[rodain-app-2.1.0_OtherOS.jar.zip](https://github.com/keeps/roda-in/files/2415328/rodain-app-2.1.0_OtherOS.jar.zip)

**Comment by luis100** [comment]
Fixed at 56e3f32cc4496c23d84655987f3a7d658bf4e5db
